### DISCUSSION TITLE
Learn Build Teach Hackathon Submissions

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#47] [News + Announcements]
As a reply to this post, submit your project using the following template:

```
## Title
### Description
{{Text for short description}}

**Link to code:**
**Link to blog post:**
```

The completed submission should look like this:

### Description
For this project, I access the microphone to power a Christmas song search using the Genius API. 

**Link to code:** https://github.com/deepgram-devs/song-search
**Link to blog post:** https://blog.deepgram.com/song-search-js/

**Comment by dotAadarsh** [comment]
## Building YouTXT
### Transcribing YouTube Videos

YouTXT is an app that transcribes any YouTube video and provides some additional features like summary, keywords, Transcript translation, search the video, and markdown editor to edit the transcription.

**Link to code:** https://github.com/dotaadarsh/YouTXT
**Project Live URL:**: https://youtxt.streamlitapp.com/
**Link to blog post:** https://aadarshkannan.hashnode.dev/building-youtxt

**Comment by Manikant92** [comment]
## VProfressor.ai — A 24x7 Virtual Professor for Students

### An application that it helps the students and increase their speed, efficiency of learning and also get their doubts clarified with additional learnings.

VProfessor.ai contains different features designed to facilitate online learning. Users can upload a wav/mp3/ text files. Then, a Deepgram STT transcript of the audio is returned, along with a summary of the data. This includes key words and main topics, Wikipedia page links, current events from NewsAPI, and recommended YouTube videos. From here, users can either read the summary report on the website or download it as a pdf for their personal studying. Additionally, students may “ask the professor” a question, and get a quick short answer for themselves which leverages wolframalpha api’s.

Link to code: https://github.com/Manikant92/VProfessor_LBTHackathon
Link to blog post: https://medium.com/@kellamanikant/vprofressor-ai-a-24x7-virtual-professor-for-students-91b01b88fee6

**Comment by Manikant92** [comment]
Can we please know when the winner results be announced?

**Reply by BekahHW** [reply]
Within a week!

**Reply by Manikant92** [reply]
Thanks Bekah

**Reply by BekahHW** [reply]
You're welcome, @Manikant92

**Reply by Manikant92** [reply]
Hello @BekahHW  Curios to know, please let us know on the tentative date for winners announcement?, Thanks!

**Reply by Manikant92** [reply]
Hi @BekahHW looking for an update from you!

**Reply by BekahHW** [reply]
@Manikant92 we'll announce it via a blog post on Friday that we'll also put in the Learn Build Teach Discord.

**Reply by Manikant92** [reply]
Thanks for the update @BekahHW

**Reply by Manikant92** [reply]
Can you please share the discord link here?

**Reply by BekahHW** [reply]
@Manikant92 you should already be in the discord to participate in the hackathon. The submission form required your LBT discord handle.

**Reply by Manikant92** [reply]
Yes, I found the channel. But don't see any update as of now.

**Reply by Manikant92** [reply]
@BekahHW

**Reply by Manikant92** [reply]
I found the blog post from deepgram blog now. Can you please let us know the criteria of selecting the winners please? Also, as per hackathon, there are 2 grand prizes and 5 runner up prizes, only one winner and runner up prizes are announced on the page. Please let us know criteria of selection? Thanks @BekahHW

**Reply by BekahHW** [reply]
Hey @Manikant92. To be a valid submission, you need to be a member of the LBT community. As per the guidelines, the project had to be created for this hackathon. Any work submitted for any other hackathons, had to be disclosed as per the agreed to terms and conditions. 

The two winners were the two valid submissions we received, and judging was based on factors including innovation, use of Deepgram, and code submission.

**Reply by Manikant92** [reply]
Thanks @BekahHW for confirmation. This is just a feedback, I came to know about this hackathon via your monthly marketing emails, however I lately realised the requirement was to be present in discord channel as well. But i couldn't find the link to join in any of your communications either email or post about this hack. So, from next time, can you please mention the relevant links in your hackathon communications so that developers who are interested to participate can join in all sources. Thanks!

**Comment by TekGadgt** [comment]
## DiscordSTT
### A discord bot with per user transcription functionality.
This started as a Virtual Table Top transcription concept, to allow virtual D&D campaigns to be more accessible, but we soon realized that most folks playing D&D on VTTs were using Discord for Voice.  So we expanded the scope to handle Discord voice.  (I don't have a blog and didn't realize this was what was meant by a github discussion, so we actually wrote our post on our repo's discussions, linked below).

**Link to code:** [https://github.com/TekGadgt/discordstt](https://github.com/TekGadgt/discordstt)
**Link to blog post:** [https://github.com/TekGadgt/discordstt/discussions/5](https://github.com/TekGadgt/discordstt/discussions/5)

**Comment by DeepPleb** [comment]
## DeepCensor
### Censor swear words automatically
Using profane language is often unwelcomed. You don't hear an explicit song on the radio. A Youtube video with swears could be demonetized. Live news with profanity might even land the channel in legal trouble. But censoring audio can be a time consuming task.

Thanks to AI we can now do this automatically. DeepCensor is a piece of python code which **takes in an audio file (or a live stream link) and spits out an output file with all the swears muted**. This process is entirely automatic thanks to the Deepgram python SDK.  

**Link to code:** https://github.com/DeepPleb/DeepCensor
**Link to blog post:** https://medium.com/@scarphaze000/deepcensor-1c402df7f270
